Samples/VirtualCamera/VirtualCameraManager_WinRT/VirtualCameraManager.idl

```diff
@@ -0,0 +1,66 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+namespace VirtualCameraManager_WinRT
+{
+    // types of virtual camera exposed by the VirtualCameraMediaSource
+    enum VirtualCameraKind
+    {
+        Synthetic = 0, // will generate frames on its own
+        HookingARealCamera = (Synthetic + 1) // will relay frames and properties from an existing camera
```

Comment:
>HookingARealCamera [](http://example.com/codeflow?start=8&length=18)

this name feels awkward

Reply:
how so

Reply:
lets use word "physical" instead of real. also "relaying"/"wrapping" instead of hooking